File: Sdl.Tridion.Context/OsClaims.cs
using Tridion.ContentDelivery.AmbientData;

namespace Sdl.Tridion.Context
{
    public class OsClaims
    {

        public OsClaims()
        {
        }

        public string Model { get { return AmbientDataContext.CurrentClaimStore.Get<string>(ClaimUris.UriOsModel); } }
        public string Variant { get { return AmbientDataContext.CurrentClaimStore.Get<string>(ClaimUris.UriOsVariant); } }
        public string Vendor { get { return AmbientDataContext.CurrentClaimStore.Get<string>(ClaimUris.UriOsVendor); } }
        public string Version { get { return AmbientDataContext.CurrentClaimStore.Get<string>(ClaimUris.UriOsVersion); } }
    }
}

File: com/sdl/tridion/context/OsClaims.java
package com.sdl.tridion.context;

import java.net.URI;
import java.util.Map;

public class OsClaims extends ContextClaims {
	public OsClaims(Map<URI, Object> claims) {
		super(claims);
		tafContext = "taf:claim:context:os:";
	}

	public OsClaims() {
		super();
		tafContext = "taf:claim:context:os:";
	}

	public String getModel() {
		return getStringValue(ClaimUris.UriOsModel);
	}

	public String getVariant() {
		return getStringValue(ClaimUris.UriOsVariant);
	}

	public String getVendor() {
		return getStringValue(ClaimUris.UriOsVendor);
	}

	public String getVersion() {
		return getStringValue(ClaimUris.UriOsVersion);
	}
}
